docs/bigquery-usage.rst

```diff
@@ -34,31 +34,74 @@ Authorization / Configuration
      >>> client = bigquery.Client()
 
 - Override the credentials inferred from the environment by passing explicit
-  ``credentials`` to one of the alternative classmethods, `from_service_account_json`:
+  ``credentials`` to one of the alternative ``classmethod`` factories,
+  `:meth:gcloud.bigquery.client.Client.from_service_account_json`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_json('/path/to/creds.json')
 
-  or `from_service_account_p12`:
+  or `:meth:gcloud.bigquery.client.Client.from_service_account_p12`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_p12('/path/to/creds.p12', 'jrandom@example.com')
 
-- Override the project inferred from the environment by passing an explicit
-  ``project`` to the constructor, or to either of the alternative classmethods:
+
+Projects
```

Comment:
Note that I have *not* introduced a `Project` class here anywhere:  it doesn't seem worthwhile, especially given that the BQ backend doesn't surface a resource representation for a project. I got here because:

- The BQ docs treat "project" as a container for dataset -- in particular, they reference the ["project roles"](https://cloud.google.com/bigquery/access-control#projectroles) as being acquired by default by datasets within the project.  We have to explain that bit in order to make the "dataset roles" comprehensible.
- The BQ backend adds a ["list projects" API](https://cloud.google.com/bigquery/docs/reference/v2/projects/list).  If we don't care about allowing for that query without knowing a valid project (@dhermes mentioned colleagues who wanted that feature in the past), we can drop the `project=None` override.